packages/react-data-grid/src/Cell/CellContent.tsx

```diff
@@ -66,9 +46,25 @@ export default function CellContent<R>({
     };
   }
 
+  function getFormattedValue(formatter: typeof column['formatter']) {
```

Comment:
Should we create a type called `Formatter<TRow, TDependentValue = unknown, TField extends keyof TRow = keyof TRow>` and assign to both column and here?

Reply:
We could, I'm just lazy and we're deprecating formatters anyway 🤷‍♂